tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = tests, flake8
```

Comment:
Instead of adding in that setup.py, you want skipsdist = true, as I did in https://github.com/mozilla/mozillians-tests/pull/249/files#diff-b91f3d5bd63fcd17221b267e851608e8R2